web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -140,7 +140,7 @@ export function FilterPanel({
           kindsFromParams={kinds || []}
         />
         {ClusterDropdown}
-        {availabilityFilter && (
+        {availabilityFilter && availabilityFilter.canRequestAll && (
```

Comment:
What if `canRequestAll` is `false`? Previously we could choose between `accessible` and `requestable`.
I think we should still allow this.

Reply:
Ok sure. I originally programmed this as "if the feature is off, just dont show it at all", but i'll udpate to still show it and use `requestable`